CONTRIBUTING.md

```diff
@@ -33,12 +33,23 @@ See the next section for how to adapt to upstream changes.
 
 ### Branching Strategy
 
-By default development happens in branches, which are merged via pull requests.
+By default, development happens in branches, which are merged via pull requests.
 Special cases, like fixing problems with the CI pipeline, are of course exempt from this guideline.
 
 Please make sure to give branches a meaningful name!
 As an example, the one creating this documentation was called `branching-merging-documentation`.
 
+### Documentation
+
+When a new extension was added (or significant changes were made) please update the documentation.
+To add the documentation for a new extension add a new entry in the `docs/docs-nav.yml` file and add a new `.adoc` file for the new extension.
```

Comment:
i think we can remove all the `new` in this line. because it also applies to existing extensions. and adding an entry implies for me that it is somehwat new :)

Reply:
```suggestion
To add documentation for an undocumented extension add an entry in the `docs/docs-nav.yml` file and create a new `.adoc` file for the extension.
```